### DISCUSSION TITLE
Поиск

### DISCUSSION THREAD

**Discussion by tipodanet** [k-ivan/visual-bookmarks-chrome#184] [Ideas]
Он есть ! Это уже  половина успеха. 
Предложение: **Настройки поиска.** 
Это тема не новая , на любом форуме можно найти десятки вариантов воплощения.
И кстати, Вивальди отслеживает  для сортировки такие параметры: 
-- Название,
-- Дата создания, 
-- Дата изменения, 
-- Вручную
-- Папки и Закладки по алфавиту
-- (прямая/обратная).

Задел хороший. В ихнем JSON это все есть. И идеале все из Сортировки можно перенести и в Настройки Поиска. 
В некоторых вопросах он лидер -- ни в одном известном мне браузере такой мощной сортировки нет . Но Поиск в Закладках у них не доделан. Он даже не дотягивает до стандартного поиска, как в Вашем расширении. Там проблема с логикой и фильтрами, этой проблеме лет 5-6 . И до разрабов не достучатся. А с пиарщиком говорить бесполезно.

Но применительно к поиску в Закладках я еще нигде не встречал подобного простейшего решения:
```
1 -- Закладки + Папки (сейчас)
2 -- Закладки
3 -- Папки
```
Иван, не сложно , правда ? 
У меня  есть видео, снятое для левого браузера. Там показана ситуация: Поиск выдает 250 Закладок и 1 Папку.  Мне нужна эта 1 Папка. И найти эту 1 Папку в результатах 251 штук -- не легко, сразу и не видно, где там папка в этом столбе результатов закопана.
Очень красиво для  обсуждения возможности разделения поиска.
Предложение на чужом форуме есть, прошло 2 года, можно приплюсовать еще 3 не меньше. Там бесполезно что-то просить, не дождешься. Люди так и пишут, "не мне  так детям"  :) 
Иван. мне интересна судьба этого предложения на вашем форуме :)

**Comment by tipodanet** [comment]
На разных форумах  настройки поиска занимают полстраницы под/над строкой Поиска. Но здесь не форум. Здесь Закладки. Именно это предложение можно сделать прямо в строке Поиска в конце в виде выпадающего списка с 3 вариантами.

**Reply by k-ivan** [reply]
Можно подумать над этим
